### DISCUSSION TITLE
Timeline plots in wave

### DISCUSSION THREAD

**Discussion by shamilh2o** [h2oai/wave#2081] [Q&A] [answered]
Hey, Is there any plot type available in wave to create this type of timeline plot?
<img width="1016" alt="image" src="https://github.com/h2oai/wave/assets/101506482/b0de16ad-7d9b-4b40-a77b-74678ee70208">

 Above plot is picked from a place which is created using "matplotlib" python package. 
 cc: @mturoci @mtanco

**Comment by mturoci** [comment, marked as answer]
This kind of plot is not supported in Wave as far as I remember. However, matplotlib is supported so you can use it in your Wave app if needed.

**Reply by shamilh2o** [reply]
Thanks @mturoci . 
In case I don't think like ui.plot_card can be used for that. 
In above plot, I saw it using matplotlib to create it and encoded it to display in an image card. Is that the way we can use matplotlib within Wave or any other hints to check in?

**Reply by mturoci** [reply]
Depends on the output of matplotlib. If it's an img, you need `image_card`. If it was HTML with embedded JS, then markup card etc.
